ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/restore/KsqlRestoreCommandTopic.java

```diff
@@ -65,17 +79,42 @@ private static KsqlConfig loadServerConfig(final File configFile) {
     return new KsqlConfig(serverProps);
   }
 
-  public static List<Pair<byte[], byte[]>> loadBackup(final File file) throws IOException {
+  public static List<Pair<byte[], byte[]>> loadBackup(
+      final File file,
+      final RestoreOptions options,
```

Comment:
nit: try and use the same variable name for the same thing everywhere (so either `options` or `restoreOptions`